src/common/components/ui/formik/utils.ts

```diff
@@ -53,32 +46,57 @@ export const parseStringLabels = (
   return labels;
 };
 
-// Input: ['foo=bar', 'key=value', 'foo=blee']
-// Result: ['key=value', 'foo=blee']
-export const uniqueLabels = (labelPairs: string[]): string[] =>
-  labelsToArray(parseStringLabels(labelPairs));
+type innerError = { path: string; message: string };
+type innerErrors = { inner: innerError[] };
+type fieldErrors = { [key: string]: string[] };
 
-// Input: (['foo=bar', 'key=value'], ['foo', 'bar'])
-// Result: ['foo=bar']
-export const selectedLabelsOnly = (labelPairs: string[], allowedKeys: string[]) =>
-  labelPairs.filter((pair) => allowedKeys.includes(pair.split('=')[0]));
+const fieldErrorReducer = (errors: innerError[]): fieldErrors => {
+  return errors.reduce(
+    (memo, { path, message }) => ({
+      ...memo,
+      [path]: (memo[path] || []).concat(message), // eslint-disable-line
+    }),
+    {},
+  );
+};
```

Comment:
we can remove the `// eslint-disable-line`
```suggestion
const fieldErrorReducer = (errors: innerError[]): fieldErrors => {
  return errors.reduce<fieldErrors>(
    (memo, { path, message }) => ({
      ...memo,
      [path]: (memo[path] || []).concat(message),
    }),
    {},
  );
};
```

Reply:
Done